benchmarks/zdc_lambda/config.yml

```diff
@@ -0,0 +1,34 @@
+simulate:zdc_lambda:
+  stage: simulate
+  extends: .phy_benchmark
+  needs: ["common:detector"]
+  parallel:
+    matrix:
+      - P: 100
+      - P: 125
+      - P: 150
+      - P: 175
+      - P: 200
+      - P: 225
+      - P: 250
+      - P: 275
+  timeout: 3 hours
+  script:
+    - snakemake --cores 1 sim_output/zdc_lambda/epic_zdc_sipm_on_tile_only_rec_lambda_dec_${P}GeV.edm4hep.root 
+  retry:
+    max: 2
+    when:
+      - runner_system_failure
+
+analyze:zdc_lambda
```

Comment:
The error reporting in github trigger action is not working, but you can be sure that issue is gitlab syntax
https://eicweb.phy.anl.gov/EIC/benchmarks/detector_benchmarks/-/pipelines/102784

Reply:
```suggestion
analyze:zdc_lambda:
```